css/style.css

```diff
@@ -80,22 +102,12 @@ li {
 }
 
 .line1 {
-  height: 0;
-  width: 350px;
-  left: 0;
-  top: 9px;
-  border-radius: 10px;
-  padding: 40px;
-  margin: auto;
+  width: 300px;
+  margin: 50px auto;
   overflow: hidden;
   justify-content: center;
 }
```

Comment:
Kindly notice that these lines are larger and thinner in the original design 😉:
| Figma design  | Your design |
| ------------- | ------------- |
| ![image](https://user-images.githubusercontent.com/42799579/199633091-78f68ebf-170e-448d-98c5-6b507d138bdf.png)  | ![image](https://user-images.githubusercontent.com/42799579/199633020-c76791cc-c33c-4269-92c7-49861764c9f6.png)  |

Tip: to make it thinner, you can update the border:
![image](https://user-images.githubusercontent.com/42799579/199633125-d251e3b2-3a8c-42c6-a53f-e956c27022df.png)

Reply:
i cant make the line thiny , it doesnt work 
![imagen](https://user-images.githubusercontent.com/90804443/199649751-aa183dde-661c-4ba7-8ff7-439b82554258.png)

Reply:
> i cant make the line thiny , it doesnt work ![imagen](https://user-images.githubusercontent.com/90804443/199649751-aa183dde-661c-4ba7-8ff7-439b82554258.png)

Try to update the border, like this:
```css
.line1 {
    border: 1px solid #646369
}
```

If it doesn't get any thinner, don't worry, the display can change from device to device. 1px is the lowest value possible.